src/org/labkey/test/components/ui/grids/GridBar.java

```diff
@@ -217,7 +217,11 @@ public GridBar clearAllSelections()
      */
     public void clickButton(String buttonCaption)
     {
-        Locator.buttonContainingText(buttonCaption).waitForElement(this, 5_000).click();
+        Locator.XPathLocator btnAsLink = Locator.tagWithClass("a", "btn").containing(buttonCaption);
+        if (btnAsLink.isDisplayed(this))
+            btnAsLink.findElement(this).click();
+        else
+            Locator.buttonContainingText(buttonCaption).waitForElement(this, 5_000).click();
```

Comment:
Just a style comment... 
Alternatively you could try this:
```
        WebElement button = Locator.tagWithClass("a", "btn").containing(buttonCaption).findWhenNeeded(this);
        waitFor(button::isDisplayed, String.format("Button with caption '%s' is not visible.", buttonCaption), 5_000);
        button.click();

```

Reply:
This change wasn't because I had to wait for the button to display. It is because sometimes the grid bar buttons are `<button>` elements and sometimes they are `<a class="btn">` elements.

Reply:
Would `BootstrapLocators.button` cover both kinds of button?

Reply:
Yes, looks like that should work. Thanks